linux/LinuxProcessList.c

```diff
@@ -1640,8 +1684,13 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
       }
       #endif
 
-      if (ss->flags & PROCESS_FLAG_LINUX_CGROUP) {
-         LinuxProcessList_readCGroupFile(lp, procFd);
+      /* The process' cgroup name is required for checking whether it runs in a container */
+      LinuxProcessList_readCGroupFile(lp, procFd);
+      char *cgroup = lp->cgroup;
+      if ( lp->cgroup == NULL
+         || (!(String_startsWith(cgroup, "/user") || String_startsWith(cgroup, "/machine") || String_startsWith(cgroup, "/system"))) ){
+         if (!LinuxProcessList_checkPidNamespace((Process *)lp, procFd))
+            goto errorReadingProcess;
```

Comment:
Not sure this should cause a full bail-out for reading this process …

@cgzones @fasterit What do you think?